### PR TITLE
Exit code improvements

### PR DESCRIPTION
**Pull request by cea2aj**
Catch and handle all exit codes at the top-level of jambo

During the localized-pages rebase, some promise rejections were unhandled and jambo was incorrectly exiting with a code of 0. This was happening because the previous exit-code architecture made the commands responsible for catching and handling thrown exceptions and unhandled promise rejections. This change removes this responsibility from the commands and instead handles all errors and unhandled rejections at the top level.

J=SLAP-796
TEST=manual

Test various Jambo commands and confirm they work and exit with the right codes. Confirm that jambo does not exit with a code of 0 for unhandled promise rejections.

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -11,11 +11,15 @@ const CommandImporter = require('./commands/commandimporter');
...
-}
+// Exit with a non-zero exit code for unhandled rejections and uncaught exceptions
+process.on('unhandledRejection', err => {
+  exitWithError(err);
+});
+process.on('uncaughtException', err => {
+  exitWithError(err);
+});
+
+jamboConfig = fs.existsSync('jambo.json') && parseJamboConfig();
```
Comment (alexisgrow) [inline src/cli.js:22]: nit: can this be a const?

**Review by alexisgrow — COMMENTED** [review]
One comment, otherwise LGTM

File: src/cli.js (hunk 1 above)
Comment (cea2aj) [reply src/cli.js:22]: sure

**Review by nanhu95 — APPROVED** [review]
LGTM!

**Review by oshi97 — APPROVED** [review]
looks great!

**Review by alexisgrow — APPROVED** [review]
LGTM!

**Pull request merged**